### PR TITLE
[13.0][ADD] New module 'account_invoice_send_mail_queued': version 13.0

### PR DESCRIPTION
**Pull request by manuelregidor**
Send emails with invoices asynchronously when using the "Send & print" invoices action and multiple invoices have been selected.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
As a better module name, I see `account_invoice_send_mail_queued`

**Comment by manuelregidor** [comment]
> As a better module name, I see `account_invoice_send_mail_queued`

done!

**Comment by manuelregidor** [comment]
@pedrobaeza The module has been renamed. Could you review it, please?

File: account_invoice_send_mail_queued/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+This module needs no special configuration. However, the module 'queue_job' needs some configuration.
```
Comment (pedrobaeza) [inline account_invoice_send_mail_queued/readme/CONFIGURE.rst]: Shouldn't this point to the `queue_job` README instructions instead?

File: account_invoice_send_mail_queued/tests/test_asynchronous_invoice_email.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+class TestAsynchronousInvoiceEmail(common.TransactionCase):
+    def setUp(self):
+        super(TestAsynchronousInvoiceEmail, self).setUp()
+        self.account_invoice_send_obj = self.env["account.invoice.send"]
+        self.job_obj = self.env["queue.job"]
+        self.invoice_obj = self.env["account.move"]
+
+    def test_delay_send_invoices(self):
+        invoice_1 = self.env.ref("l10n_generic_coa.demo_invoice_1").id
```
Comment (pedrobaeza) [inline account_invoice_send_mail_queued/tests/test_asynchronous_invoice_email.py]: Don't hardcode this. You are forcing to have `l10n_generic_coa` installed, which is not the case on integration CIs.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Remove .DS_Store files

File: account_invoice_send_mail_queued/readme/CONFIGURE.rst (hunk 1 above)
Comment (manuelregidor) [reply account_invoice_send_mail_queued/readme/CONFIGURE.rst]: Done

File: account_invoice_send_mail_queued/tests/test_asynchronous_invoice_email.py (hunk 2 above)
Comment (manuelregidor) [reply account_invoice_send_mail_queued/tests/test_asynchronous_invoice_email.py]: Done

**Comment by manuelregidor** [comment]
Hi @pedrobaeza Could you review, please?

**Review by pedrobaeza — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
LGTM

**Comment by pedrobaeza** [comment]
Please rebase and fix pre-commit stuff.

**Comment by manuelregidor** [comment]
> Please rebase and fix pre-commit stuff.

Could you give me more details on how to proceed? Sorry, but I'm not sure what is the next step. Thank you.

**Comment by pedrobaeza** [comment]
Please ask @ValentinVinagre about how to rebase you branch over current 13.0 OCA branch, and run pre-commit stuff.

**Comment by pedrobaeza** [comment]
Incorrect rebasing, probably because you haven't fetched latest remotes.

**Comment by HaraldPanten** [comment]
Hi @pedrobaeza ,

This one (the original one, not the incorrect rebased one) isn't the same (or quite similar) to https://github.com/OCA/account-invoicing/pull/770/files ?

THX

**Comment by pedrobaeza** [comment]
May it be. I didn't know that one!

**Comment by HaraldPanten** [comment]
Us neither. We'll close this one, then

**Pull request closed without merging**